### DISCUSSION TITLE
request help: I can't use print() method in lua, when I use print("test"),there is no output in log files and console.

### DISCUSSION THREAD

**Discussion by hizzyishome** [apache/apisix#8107] [Q&A]
### Current Behavior

 I can't use print() method in lua, when I use print("test"),there is no output in log files and console.

### Expected Behavior

I dont want the begaining like "2022/10/18 09:57:29 [warn] 65#65: *262322 [lua] ****.lua:114:" or the end like " client: 127.0.0.1, server: _, request: "POST /123 HTTP/1.1", host: " .
I need pure formated log to be printed.
user print("test")
I can get the line:
test

that's all.

### Steps to Reproduce

1. at any lua script running lifetime
2. print("test")
3. tail error.log and access.log there is no "test" print

### Environment

- APISIX version (run `apisix version`): 2.15.1 2.12.1both
- Operating system (run `uname -a`):inux  4.12.14-120-default #1 SMP Thu Nov 7 16:39:09 UTC 2019 (fd9dc36) x86_64 x86_64 x86_64 GNU/Linux
- OpenResty / Nginx version (run `openresty -V` or `nginx -V`):openresty/1.19.9.1
- etcd version, if relevant (run `curl http://127.0.0.1:9090/v1/server_info`):
- APISIX Dashboard version, if relevant:
- Plugin runner version, for issues related to plugin runners:
- LuaRocks version, for installation issues (run `luarocks --version`):3.8.0

**Comment by tzssangglass** [comment]
> * at any lua script running lifetime
> * print("test")
> * tail error.log and access.log there is no "test" print

1. you can use print to output before APISIX starts.
2. after APISIX starts, the error log and access log are take over by Nginx's logging system, which you can output with `core.log.info|ddebug|error|...`


> I dont want the begaining like "2022/10/18 09:57:29 [warn] 65#65: *262322 [lua] ****.lua:114:" or the end like " client: 127.0.0.1, server: _, request: "POST /123 HTTP/1.1", host: " .
> I need pure formated log to be printed.

you can turn off the access log, raise the log level of the error log, and then use the log plugin to collect logs, this allows you to decide the format and content of the log.
